datacube_stats/statistics.py

```diff
@@ -798,7 +798,10 @@ def compute(self, data):
 
             # Grab a copy of the coordinates we need for creating the output DataArray
             output_coords = copy(squashed.coords)
-            del output_coords['time']
+            if 'time' in output_coords:
+                del output_coords['time']
+            if 'source' in output_coords:
+                del output_coords['source']
 
```

Comment:
maybe it's time to take this out into a function and generalise it accept a list of coords to "forget", also should this be a blacklist of whitelist?